app/models/contact_wct.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ContactWct < ContactForm
+  attribute :message, validate: true
+
+  def to_email
+    "contact@worldcubeassociation.org"
+  end
+
+  def subject
+    Time.now.strftime("[WCA Website] General Comment by #{name} on %d %b %Y at %R")
+  end
+end
```

Comment:
Where was this done previously? How can the current form work without this file?

Reply:
This was done in website_contact.rb. I didn't remove it from that file because I am thinking to remove that file completely and create a model for each recipient separately once this PR is merged.